sources/packages/api/src/route-controllers/assessment/assessment.controller.service.ts

```diff
@@ -273,4 +287,124 @@ export class AssessmentControllerService {
       });
     return finalAward;
   }
+
+  /**
+   * Get all requests related to an assessments for a student
```

Comment:
`an assessments`?

Reply:
The comment was copied from the controller method. But I fixed the description in all places.